kotlintest-assertions/src/jvmTest/kotlin/com/sksamuel/kotlintest/properties/GenTest.kt

```diff
@@ -602,6 +606,12 @@ class GenTest : WordSpec() {
         }
       }
     }
+
+    "Gen.samples(sampleValues)" should {
```

Comment:
Could you please add some more scenarios? Such as verifying that the order is correct, and verifying that the constants are exactly the same as passed in.

```
val gen = Gen.samples(1, 2, 3, 4, 5)
gen.random().take(10).toList() shouldBe (1, 2, 3, 4, 5, 1, 2, 3, 4, 5)
```